reload_app/events.py

```diff
@@ -263,6 +263,12 @@
     "dashboards_views.edit_widget_modal.confirm": {
         "org_id": int,
     },
+    "dashboards_views.widget_resize": {
```

Comment:
```suggestion
    "dashboards_views.widget.resize": {
```

@narsaynorath would this naming scheme make more sense? Then you should extend to `dashboards_views.widget.<other actions>`.

Reply:
Gah! lol that's what I was using in my other PR
https://github.com/getsentry/sentry/pull/31750/files#diff-d04aeb99ab26e69d6d25c1f3e822e96c36b0dd336b46c7eea3f3498b4497727aR81

Good catch, I appreciate it